src/wallet/wallet.h

```diff
@@ -475,7 +475,34 @@ class CWalletTx : public CMerkleTx
 };
 
 
-
+class CInputCoin {

```

Comment:
It is not visible on github but local diff shows me that line endings are weird here:
```diff
-using CInputCoin = std::pair<const CWalletTx*, unsigned int>;
+class CInputCoin {^M
+public:^M
```
^M is 13, so I suppose these are DOS line endings. Looks like it is only in `src/wallet/wallet.h`, and only the newly  introduced code.
suggest running:
```bash
$ dos2unix src/wallet/wallet.h
dos2unix: converting file src/wallet/wallet.h to Unix format ...
```